.github/workflows/ci.yml

```diff
@@ -119,7 +119,7 @@ jobs:
         uses: dtolnay/rust-toolchain@stable
         with:
           # cargo check-external-types is only available on this specific nightly
-          toolchain: nightly-2024-05-01
+          toolchain: nightly-2024-06-30
```

Comment:
Can you move this into its own fix?

Reply:
Yes, it can be done.

Reply:
See https://github.com/open-telemetry/weaver/pull/442